src/code/code_800C3C20.c

```diff
@@ -0,0 +1,12 @@
+#include <global.h>
+
+u8 D_8012D200[] = {
+    0, 1, 2, 3, 4, 5, 6,
+};
+
+void func_800C3C20() {
+    u8* iter;
+    for (iter = D_8012D200; iter < D_8012D200 + 7; iter++) {
+        func_800F87A0(*iter);
+    }
```

Comment:
Could the loop be written like this?
```suggestion
    for (i = 0; i < 7; i++) {
        func_800F87A0(D_8012D200[i]);
    }
```

Reply:
This didn't match for some reason. (There was an issue with the compare if I recall correctly).

Reply:
There must be way to write it without using the address of D_8012D200...
What about if it's a single line?